schema/crdb/README.adoc

```diff
@@ -58,13 +61,20 @@ Process:
 
 * Choose a `NEW_VERSION` number. This should almost certainly be a major
   version bump over `OLD_VERSION`.
-* Add a file to `schema/crdb/NEW_VERSION/up.sql` with your changes to the
-  schema.
-** This file should validate the expected current version transactionally.
```

Comment:
> This file should validate the expected current version transactionally.

I'm not sure what this means.  I didn't think anything like this should be necessary so I removed it, but I'd like a +1 that I'm not missing something important!

Reply:
You are correct — it used to be the case ([example](https://github.com/oxidecomputer/omicron/blob/cec576fc6b3c56e782e655465340e050636d7cb2/schema/crdb/3.0.0/up.sql)) that schema changes had to contain an explicit `BEGIN` and `COMMIT`, but now the migration doer handles wrapping each `up*.sql` in its own transaction.

Reply:
I agree with @david-crespo -- all the more reason to update these docs.